### ISSUE TITLE
Deeplink chat message does not get removed from inputfield after sending

### ISSUE THREAD

**Issue by sirLoaf** [MicrosoftDocs/msteams-docs#8106]
Hi Folks

I'm currently using the following deeplink
`https://teams.microsoft.com/l/chat/0/0?users=<userUPN>&message=some%20random%20text`

It opens up Teams (Desktop) or can be used in the web. The provided message gets prefilled into the inputfield and I can successfully send the message.

The problem is, that after sending the message does not get cleared from the input field.
[IssueGif](https://github.com/sirLoaf/dump/blob/main/chrome_Hkq7AuQHNy.gif?raw=true)
Any help would be appreciated, thanks!

**Comment by Nivedipa-MSFT** [comment]
@sirLoaf - Thanks for reporting your issue. We will test this and update you soon.

**Comment by Nivedipa-MSFT** [comment]
@sirLoaf - We are able to repro your issue. We have raised a bug for the same. We will let you know once we get any updates on it.

**Comment by sirLoaf** [comment]
@Nivedipa-MSFT Please keep me posted about any updates, thanks.
Another question/request from my side: Is there any chance/plan to support linebreaks also in this &message parameter? So far I have not been able to achieve that. Thanks

**Comment by Nivedipa-MSFT** [comment]
@sirLoaf - Could you please raise a [feature request](https://feedbackportal.microsoft.com/feedback/forum/ad198462-1c1c-ec11-b6e7-0022481f8472) for supporting linebreaks in this &message parameter.

**Comment by Nivedipa-MSFT** [comment]
@sirLoaf - We have tested the provided deeplink on Teams web client and desktop client version 1.6.00.19571, and we are not able to reproduce the issue now. Could you please update to the latest Teams version and test this again? Please confirm if the problem is resolved.

The deeplink to test is: https://teams.microsoft.com/l/chat/0/0?users=<userUPN>&message=some%20random%20text

**Comment by sirLoaf** [comment]
@Nivedipa-MSFT yes, it's not reproducible anymore. Thanks and sorry for the delay
